core/store/src/main/java/uk/gov/gchq/gaffer/store/StoreProperties.java

```diff
@@ -461,4 +461,25 @@ public void updateStorePropertiesClass(final Class<? extends StoreProperties> re
 
         return properties;
     }
+
+    public static StoreProperties resolveStoreProperties(final Store store, final StoreProperties properties, final String parentStorePropertiesId) {
```

Comment:
This doesn't really feel like the right place to put this method. It creates a dependency both ways between store and StoreProperties. Perhaps it could just go in GraphLibrary. Similar with resolveSchema.